### PR TITLE
Multisite compat & bug fixes

### PR DESCRIPTION
**Pull request by roborourke**
- Ensure AUTH_KEY is defined or read from site options / options
- Ensure correct namespace for accelerate version
- Allow preview thumbs for xb post type as well, subset of reusable blocks

### CODE REVIEW

File: inc/dashboard/namespace.php [hunk 1]
```diff
@@ -146,7 +145,7 @@ function render_page() {
 function get_plugin_version() : string {
 	// Only show version if this is embedded in the accelerate plugin.
 	if ( defined( 'Altis\\Accelerate\\VERSION' ) ) {
-		return Altis\Accelerate\VERSION;
+		return \Altis\Accelerate\VERSION;
 	}
```
Comment (robindevitt) [inline inc/dashboard/namespace.php:149]: Why would the `return` be "`\Altis\Accelerate\VERSION;`" but the `if` statement is "`Altis\\Accelerate\\VERSION`"

**Review by shadyvb — APPROVED** [review]

**Review by robindevitt — COMMENTED** [review]
Just a single question from my based on the one change

File: inc/dashboard/namespace.php (hunk 1 above)
Comment (shadyvb) [reply inc/dashboard/namespace.php:149]: Should be `Altis\Accelerate\VERSION` i think, I missed that, good shout!

Comment (roborourke) [reply inc/dashboard/namespace.php:149]: In the if statement it's escaped using double backslashes as we're checking for the existence of a constant by name as a string. In that instance the `defined()` function isn't scoped to the current namespace. Adding the preceding backslash before the `return` statement is just to make absolutely certain we're reading from the root of the namespaces.

Comment (robindevitt) [reply inc/dashboard/namespace.php:149]: Thanks @roborourke

**Pull request merged**